File: src/main/java/org/soraworld/lottery/listener/EventListener.java
package org.soraworld.lottery.listener;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;
import org.bukkit.event.player.PlayerQuitEvent;
import org.soraworld.lottery.config.Config;

public class EventListener implements Listener {

    private final Config config;

    public EventListener(Config config) {
        this.config = config;
    }

    @EventHandler
    public void onPlayerJoin(final PlayerJoinEvent event) {

    }

    @EventHandler
    public void onPlayerQuit(PlayerQuitEvent event) {
    }

}

File: src/main/java/org/soraworld/lottery/listener/EventListener.kt
package org.soraworld.lottery.listener

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent
import org.bukkit.event.player.PlayerQuitEvent
import org.soraworld.lottery.config.Config

class EventListener(private val config: Config) : Listener {

    @EventHandler
    fun onPlayerJoin(event: PlayerJoinEvent) {

    }

    @EventHandler
    fun onPlayerQuit(event: PlayerQuitEvent) {
    }

}
